scripts/import_sqlite.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+"""
+Opendata csv to sqlite3 importer.
+This script allows to convert csv-stream to rows in sqlite table
+"""
+
+import argparse
+import csv
+import itertools
+import logging
+import os
+import sqlite3
+import sys
+from collections import (
+    defaultdict,
+    OrderedDict,
+)
+
+INSERT_CHUNK_SIZE = 1
+
+
+def ensure_table_exists_stmt(name, columns):
+    return 'CREATE TABLE IF NOT EXISTS {} ({})'.format(
+        name,
+        ', '.join(' '.join(item) for item in columns.items())
+    )
+
+
+def filter_row(data, known, wanted):
+    return OrderedDict({
+        key: data[key]
+        for key in known
+        if key in wanted
+    })
+
+
+def bulk_insert_stmt(table_name, keys, rows):
```

Comment:
Reworked this with `cursor.executemany()` and generator
see [its documentation](https://docs.python.org/3.5/library/sqlite3.html#sqlite3.Cursor.executemany)